01week/datatypes.js

```diff
@@ -0,0 +1,55 @@
+console.log(Date())
+
+
+
+// #3 Write a JavaScript program to convert a string to the number
+const numberToString = (str) => {
+  return parseInt(str)
+}
+
+console.log(numberToString("9"))
+
+// Write a JavaScript program to display the current day and time.
+console.log(Date())
+// Write a JavaScript program to convert a number to a string.
+var num = 77 
+num.toString()
+// Write a JavaScript program to convert a string to the number.
+Number('55556666')
+// Write a JavaScript program that takes in different datatypes and prints out whether they are a:
+// Boolean
+console.log(typeof (10 > 9))
+// Null
+console.log(typeof (null))
+// Undefined
+console.log(typeof (undefined))
+// Number
+console.log(typeof (10))
+// NaN
+// console.log(typeof ( ))
+// String
+console.log(typeof ('10'))
+// Write a JavaScript program that adds 2 numbers together.
+var x = 3
+var y = 4
+x + y
+// Write a JavaScript program that runs only when 2 things are true.
+var letter = "b"
+var num = 5
+if (letter === "b" && num === 5) {
+  text = "it is running";
+  console.log(text)
+}
+// Write a JavaScript program that runs when 1 of 2 things are true.
+var letter = "q"
+var num = 4
+if (letter === "p") || num === 3 {
+  console.log(text)
+}
+// Write a JavaScript program that runs when both things are not true.
+var letter = "q"
+var num = 4
+if (letter != "p" && num !=5) {
+  text = "not q or 4"
+  console.log(text)
+}
```

Comment:
Thank you for keeping the comments in the code, it makes it easy to read! If you make corrections email me.